From 3ccc6780ed774c48d59e50596d7ea000ce95f8d0 Mon Sep 17 00:00:00 2001
From: Felix Radensky <felix.r@variscite.com>
Date: Wed, 15 Jul 2020 17:59:40 +0300
Subject: [PATCH 05/31] serial: mxc: Make mxc_base modifiable by external code

Required when several boards use the same board header file
but use different UARTs as console.
---
 drivers/serial/serial_mxc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/serial/serial_mxc.c b/drivers/serial/serial_mxc.c
index 3efab1fa71..53a741352f 100644
--- a/drivers/serial/serial_mxc.c
+++ b/drivers/serial/serial_mxc.c
@@ -184,7 +184,7 @@ static void _mxc_serial_setbrg(struct mxc_uart *base, unsigned long clk,
 #error "define CONFIG_MXC_UART_BASE to use the MXC UART driver"
 #endif
 
-#define mxc_base	((struct mxc_uart *)CONFIG_MXC_UART_BASE)
+struct mxc_uart *mxc_base = ((struct mxc_uart *)CONFIG_MXC_UART_BASE);
 
 static void mxc_serial_setbrg(void)
 {
-- 
2.25.1

